app/experimenter/docs/adrs/0005-doc-hub.md

```diff
@@ -0,0 +1,151 @@
+# Use [TBD] for the Nimbus User Doc Hub
+
+- Status: proposed
+- Deciders: Lauren Zugai, Kate Hudson, [TBD]
+- Date: 2021-02-16
+
+Technical Story: [Documentation Hub for Nimbus Users](https://jira.mozilla.com/browse/EXP-860)
+
+## Context and Problem Statement
+
+In scaling Nimbus for 2021 and beyond, we need to provide experiment owners, data scientists, and desktop and Fenix engineers documentation to better streamline working with the platform. Following the same line of thought in [`nimbus-shared` ADR 0001](https://github.com/mozilla/nimbus-shared/blob/main/docs/adr/0001-one-console-app.md), users should expect a single source of truth for their documentation needs.
+
+## Decision Drivers
+
+- Docs should be kept in a central location
+- Finding relevant documentation should be easy, quick, and clear for all of our users, and cover their specific doc needs
+- Docs should be easily editable by anyone; maintenance should be straightforward and scalability should be taken into account
+- Hosting and deployment efforts should be very minimal
+
+## Considered Options
+
+- Mana / Confluence
+- Docusaurus + GH Pages
+- [Experimenter repo](https://github.com/mozilla/experimenter/) Wiki
+- Storybook
```

Comment:
I think `md-book` is another possible consideration we might want to look at considering it's extensively used by data/glean https://docs.telemetry.mozilla.org/concepts/glean/glean.html

Reply:
fwiw this seems at least superficially pretty similar to Docusaurus to me, in that they're both static site generators that compile markdown to docs, except that the styling isn't as nice